spec/fixtures/coffee.coffee

```diff
@@ -13,6 +13,7 @@ class quicksort
       if current < pivot
         left.push(current)
       else
+        # coffeelint: disable=no_trailing_semicolons
```

Comment:
I think we probably want to omit all `spec/fixtures` files from any linting.

I'm guessing adding this line might cause some specs to fail that may be targeting specific lines in this file by number.